__tests__/pet.test.js

```diff
@@ -0,0 +1,135 @@
+const Pet = require('../src/pet');
+
+describe('Pet', () => {
+    it('sets the name property', () => {
+        const pet = new Pet('Fido');
+        expect(pet.name).toEqual('Fido');
+    });
+});
+
+describe('isAlive', function () {
+    it('check if pet is dead', () => {
+        const max = new Pet('max');
+        max.hunger = 10;
+        max.fitness = 0;
+        max.age = 35;
+        max.isAlive;
+        expect(max.isAlive).toBe(false);
+    })
+});
+
+describe('growUp', () => {
+    it('has a initial age of 0', () => {
```

Comment:
This test should be in the constructor group since it relates to the initial creation of the pet